stdlib/LinearAlgebra/src/bunchkaufman.jl

```diff
@@ -59,13 +59,13 @@ julia> A = [1 2; 2 3]
  2  3
 
 julia> bkfact(A)
-Base.LinAlg.BunchKaufman{Float64,Array{Float64,2}}
+LinearAlgebra.BunchKaufman{Float64,Array{Float64,2}}
```

Comment:
This is much prettier IMO, rather awkward with the `Base.` prefix before :slightly_smiling_face: